deploy/kubeturbo_yamls/turbo-admin.yaml

```diff
@@ -3,33 +3,57 @@ kind: ClusterRole
 metadata:
   name: turbo-cluster-admin
 rules:
+  - apiGroups:
+      - ""
+      - batch
+    resources:
+      - pods
+      - jobs
+    verbs:
+      - '*'
   - apiGroups:
       - ""
       - apps
+      - apps.openshift.io
       - extensions
+      - machine.openshift.io
     resources:
-      - nodes
-      - pods
       - deployments
       - replicasets
       - replicationcontrollers
+      - statefulsets
+      - daemonsets
+      - deploymentconfigs
+      - machinesets
     verbs:
-      - '*'
+      - get
+      - list
+      - watch
+      - update
+      - patch
```

Comment:
Do use patch api call for any resource? There is no harm in keeping it, but if we are listing exact capabilities, I think this can be removed.

Reply:
As explained in #622, it doesn't hurt to keep the `patch` verb.